lib/hypernova/batch_renderer.rb

```diff
@@ -39,9 +39,13 @@ def initialize(jobs)
   def render(response)
     fmt_response = response.each_with_object({}) do |array, hash|
       name_of_component = array[0]
-      hash[name_of_component] = extract_html_from_result(name_of_component, array[1])
+      hash[name_of_component] = ensure_has_html(name_of_component, array[1])
+    end
+
+    after_response(fmt_response, response).each_with_object({}) do |array, hash|
+      name_of_component = array[0]
+      hash[name_of_component] = array[1]['html']
```

Comment:
Makes sense, although we should probably also update the `after_response` and our plugin to return a value that is spec compliant. ` { componentName: htmlString }`

That could be done in a future pr.